coverage/.resultset.json

```diff
@@ -1,17 +1,10 @@
 {
```

Comment:
@wolffles please add your /coverage/* folder to your .gitignore. I'd prefer not to see these files.